clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/description/DestroyCloudFoundryServiceDescription.java

```diff
@@ -16,11 +16,23 @@
 
 package com.netflix.spinnaker.clouddriver.cloudfoundry.deploy.description;
 
+import com.netflix.spinnaker.clouddriver.cloudfoundry.model.CloudFoundrySpace;
+import java.util.Collection;
+import java.util.Collections;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class DestroyCloudFoundryServiceDescription extends AbstractCloudFoundryServiceDescription {
+public class DestroyCloudFoundryServiceDescription
+    extends AbstractCloudFoundryServerGroupDescription {
   private String serviceInstanceName;
+  private CloudFoundrySpace space;
+  private boolean removeBindings;
+  private String application;
```

Comment:
We can use the *serverGroupName* (from AbstractCloudFoundryDescription) instead adding a new attribute here.

Reply:
Unfortunately this stage doesn't add that because it's not operating on a particular server group. However, we still want to validate that it has auth to do the work at the app level. This field is being added with the associated deck PR